apps/dotcms-ui/src/app/portlets/shared/dot-content-types-listing/dot-content-types-listing.module.ts

```diff
@@ -8,14 +8,19 @@ import { DotListingDataTableModule } from '@components/dot-listing-data-table';
 import { DotBaseTypeSelectorModule } from '@components/dot-base-type-selector';
 import { DotAddToBundleModule } from '@components/_common/dot-add-to-bundle';
 import { DotPipesModule } from '@pipes/dot-pipes.module';
+import { DynamicDialogModule } from 'primeng/dynamicdialog';
+import { DotContentTypeCloneDialogModule } from '@components/_common/dot-content-type-clone-dialog';
```

Comment:
This is not common, this should go into the content types components. @hmoreras can you point him here?

Reply:
Moved inside `apps/dotcms-ui/src/app/portlets/shared/dot-content-types-listing`, removed the module and declarared in `DotContentTypesListingModule`